HospitalSystem.ConsoleUI/Secretary/CrudUI.cs

```diff
@@ -0,0 +1,311 @@
+        Console.Clear();
+        UserService us = _hospital.UserService;
+
+        System.Console.WriteLine("Enter the following data: ");
+        System.Console.Write("email >> ");
+        string? email = Console.ReadLine();
+        if (email is null)
+        {
+            throw new NullInputException("Null value as input");
+        }
+        System.Console.Write("password >> ");
+        string? password = Console.ReadLine();
+        if (password is null)
+        {
+            throw new NullInputException("Null value as input");
+        }
+        System.Console.Write("first name >> ");
+        string? firstName = Console.ReadLine();
+        if (firstName is null)
+        {
+            throw new NullInputException("Null value as input");
+        }
+        System.Console.Write("last name >> ");
+        string? lastName = Console.ReadLine();
+        if (lastName is null)
+        {
+            throw new NullInputException("Null value as input");
+        }
+        System.Console.WriteLine(email);
+        if(email == "back"){
+            Console.Clear();
+            System.Console.WriteLine("Returning...");
+        }
+        else if(password == "back"){
+            Console.Clear();
+            System.Console.WriteLine("Returning...");
+
+        }
+        else if(firstName == "back"){
+            Console.Clear();
+            System.Console.WriteLine("Returning...");
+
+        }
+        else if(lastName == "back"){
+            Console.Clear();
+            System.Console.WriteLine("Returning...");
+        }
+        else{
+            Console.Clear();
+            Patient patient = new Patient(email, lastName, new MedicalRecord());
+            _hospital.PatientRepo.AddOrUpdatePatient(patient);
+            // ur.AddOrUpdateUser(new User(email, password,patient,Role.PATIENT));
+            us.Upsert(new User(email, password,patient,Role.PATIENT));
+        }
+    }
+
+    public void ReadUserPatient(){
+        Console.Clear();
+        UserService us = _hospital.UserService;
+        
+        System.Console.Write("Enter the user mail to view his data: ");
+        string? email = Console.ReadLine();
+        if (email is null)
+        {
+            throw new NullInputException("Null value as input");
+        }
+        var user = us.Get(email);
+        Patient pat = _hospital.PatientRepo.GetPatientById((ObjectId) user.Person.Id);
+        System.Console.WriteLine("Email : " + user.Email.ToString());
+        System.Console.WriteLine("Password : " + user.Password.ToString());
+        System.Console.WriteLine("First Name : " + pat.FirstName);
+        System.Console.WriteLine("Last Name : " + pat.LastName);
+        System.Console.WriteLine("");
+        System.Console.Write("Type back to get to menu: ");
+        string? back = Console.ReadLine();
+
+        while(back != "back"){
+        Console.Clear();
+        System.Console.Write("Type back to get to menu: ");
+        back = Console.ReadLine();
```

Comment:
Indent. Also does this part  just force the user to type back? Why not just let him go after reading anything?